### ISSUE TITLE
Cosmetic fix for Firefox is breaking grid?!

### ISSUE THREAD

**Issue by APusch-Tenno** [danheron/Heron.MudCalendar#118]
hello,

I used the unchanged download for version 1.1.2 and replaced the specific files with those from the fix for Firefox, but now any grid is showing incorrectly:

![grafik](https://github.com/danheron/Heron.MudCalendar/assets/107390850/a7ce35f1-9609-4491-bd49-93fdabe2c38d)
![grafik](https://github.com/danheron/Heron.MudCalendar/assets/107390850/39f4f7e6-dd7f-4ba3-a7de-6dcd3bfc7cf9)

This also happens in Chrome.
What am I missing here?

**Comment by danheron** [comment]
It looks like you are missing some styles.  Did you replace the .scss files?

**Comment by APusch-Tenno** [comment]
I checked version 1.1.2 before applying the fix on it and it displayed fine (except the usual issue in Firefox of course).
Cleared the cache, same result.
If there was a cached version of that site, it would display as before.

Are you sure all changed files are uploaded to GitHub?

**Comment by danheron** [comment]
Yes, all changed files are uploaded to Github.

You'll have to use the browser dev tools to try and find out why it's not working.  Have a look at the mud-cal-grid and mud-cal-grid-header classes.

**Comment by APusch-Tenno** [comment]
I don't know what went wrong there.
When I did it a second time with the unchanged 1.1.2 the grid looked different (but still not correct, since it had no grid lines).
Maybe while comparing the files with WinMerge something got messed up.

Then I downloaded all files from the fix again and applied them again to the unchanged 11.2 and it worked.

The only problem I have now is, that I added a work week/5 day week, which is now being shifted like someone took out the weekends from the grid, but I basically only copied the 7 day week and changed some values. This change worked fine before the fix.

I am hoping the person in the other issue about the work week will provide a pull request which will work with the Firefox fix.

Thanks for your time.
